app/views/home/_wiki_ed_impact.html.haml

```diff
@@ -27,36 +27,42 @@
     %h2.h1 Impact since 2010:
 
     %h3 Wiki Education Courses
-    %h4 6,250
+    %h4= @stats['wiki_edu_courses']
 
     %div
       .col
         %h3 Students
-        %h4 126,000
+        %h4= @stats['students']
 
       .col
         %h3 Articles worked on
-        %h4 141,000
+        %h4= @stats['worked_articles']
 
     %div
       .col
         %h3 Words added
-        %h4 106 Million
+        %h4= @stats['added_words']
         %p
-          The number of words students have donated to the world. That’s 361,000 pages – the equivalent of 77 volumes of a printed encyclopedia, shared with everyone, for free.
+          The number of words students have donated to the world. That’s 
+          = @stats['total_pages']
+          pages – the equivalent of 
+          = @stats['volumes']
+          volumes of a printed encyclopedia, shared with everyone, for free.
 
       .col
         %h3 Article views, in a single term
-        %h4 438 Million
+        %h4= @stats['article_views']
```

Comment:
This should probably in `article_views_in_millions`, because it's used twice but once is capitalized and once is not. In the current version, including "Millions" in the value results in "438 Million million" below.

Reply:
okay, 'article_views' now only hold the value in number eg: 438 which will remain same for both and i have appended Million and million after it accordingly.